app/views/static_pages/privacy.html.haml

```diff
@@ -3,50 +3,63 @@
     %h1 Privacy Policy
 
     %p
+      We use technologies like cookies (small files stored on your browser), web beacons, or unique device identifiers to identify your computer or device so we can deliver a better experience. Our systems also log information like your browser, operating system and IP address.
 
-      We use technologies like cookies (small files stored on your
-      browser), web beacons, or unique device identifiers to identify
-      your computer or device so we can deliver a better
-      experience. Our systems also log information like your browser,
-      operating system and IP address.
+    %p
+      The Swap My Vote Application collects some Personal Data from its Users including voting preferences, constituency, name, profile picture, social network IDs, email address, and mobile phone number.
 
     %p
+      The Application uses the phone number and email address to authenticate you. It uses the email address to contact you about your vote swaps and vote swapping partners. It may use the phone number to contact you during the election to confirm a swap or alert you to potential voting partners waiting for confirmation.
 
-      We also may collect personally identifiable information that you
-      provide to us, such as your name, location and email address. If
-      authorized by you, we may also access profile and other
-      information from services like Facebook and Twitter.
+    %p
+      At no time are your phone number or email address shown to anyone else. The other pieces of personal information help the Application to suggest prospective voting partners, and help users choose vote swapping partners.
 
     %p
+      In doing this the Application may show prospective vote swapping partners your stated voting preferences, constituency, profile picture, and the social network(s) you used to log in.
 
-      Our systems may associate this personal information with your
-      activities in the course of providing service to you (such as
-      pages you view or things you click on or search for).
+    %p
+      Part of your name is hidden and part of your name is shown, with the aim of helping establish trust between vote swapping partners.
```

Comment:
This is #47 but we haven't implemented it yet, so it shouldn't be included in this PR but rather held back and put in the same PR which implements #47.